db/seeds.rb

```diff
@@ -7,49 +7,73 @@ def create_user!(name)
   )
 end
 
+def create_developer!(attributes, avatar_name:)
+  developer = Developer.new(attributes)
+  developer.avatar.attach(io: File.open(Rails.root.join("test/files/#{avatar_name}")), filename: avatar_name)
```

Comment:
We already have some images in `test/fixtures/files/` – I'd rather use those instead of adding new ones.

Reply:
So just `lovelace.jpg` for all the devs, and basecamp for the company? or?

Reply:
Added the changes :)